### ISSUE TITLE
idle_timeout is not work ?

### ISSUE THREAD

**Issue by spinachmedia** [eclipse/mosquitto#417]
This report is for the version 1.4.11.
which i build by myself on CentOS 6.6.

mosquit configuration idle_timeout is not work...?

I set the following settings.
mosquitto.conf
```
log_type all
connection AtoB
address 192.168.101.11
topic /A/# out 2
start_type lazy
threshold 10
idle_timeout 10
```

but, I did not seem to have lost connection after several minutes.
start_type  and threshold  is correct work.

**Comment by ralight** [comment]
Sorry for the delay replying, I've made a change that I think fixes this. If you want to try it out, you can get it from the fixes branch.

**Comment by ralight** [comment]
I believe this is fixed in 1.4.13.
